awx/main/dispatch/worker/callback.py

```diff
@@ -123,6 +163,8 @@ def perform_work(self, body):
                         job_identifier = body[key]
                         break
 
+                self.last_event = f'\n\t- {cls.__name__} for #{job_identifier} ({body.get("event", "")} {body.get("uuid", "")})'  # noqa
```

Comment:
![Kapture 2020-09-22 at 18 07 21](https://user-images.githubusercontent.com/214912/93942489-8024a780-fcfe-11ea-9faa-7eacf12e2cfe.gif)

👀 @gamuniz @chrismeyersfsu @AlanCoding @ghjm @jakemcdermott

Reply:
This leaves me wanting more. Having a snapshot of the last job event processed is interesting but I suppose I want a log of all events processed. Not saying this is the place for that. But maybe this could be _more_ of a summary. Last 10-20 processed?